Bug: error when running pytest

When I run `python setup.py test` I get the following error:
```
______________________________________________________ ERROR collecting shapepipe/tests/test_pipeline.py _______________________________________________________
shapepipe/tests/test_pipeline.py:14: in <module>
    from shapepipe.pipeline import *
E   AttributeError: module 'shapepipe.pipeline' has no attribute 'file_iojob_handler'
```
Looks like a missing space or newline, but I could not find it.

--- a/shapepipe/pipeline/__init__.py
+++ b/shapepipe/pipeline/__init__.py
@@ -14,7 +14,7 @@ __all__ = [
     'dependency_handler',
     'execute',
     'file_handler',
-    'file_io'
+    'file_io',
     'job_handler',
     'mpi_run',
     'run_log',

